src/features/events_list/components/FullState/FullState.tsx

```diff
@@ -31,7 +22,7 @@ export function FullState({
     return eventsList.findIndex((event) => event.eventId === currentEventId);
   }, [currentEventId, eventsList]);
 
-  const handleFocus = useCallback(() => {
+  const scrollToCurrentEvent = useCallback(() => {
```

Comment:
Unused function, maybe remove to `src/features/events_list/components/EventsPanelSettings/EventsPanelSettings.tsx`

Reply:
Removed it altogether